### PR TITLE
Remove unused deps

### PR DESCRIPTION
**Pull request by jochem-brouwer**
I noticed in our README that we had a tx -> evm dependency and was confused, since evm should not depend on tx (which also is not the case)

This PR removes those dependencies and updates the dependency graph.

### CODE REVIEW

**Review by acolytec3 — APPROVED** [review]
LGTM

**Review by acolytec3 — APPROVED** [review]

File: packages/client/package.json [hunk 1]
```diff
@@ -67,11 +67,9 @@
     "@ethereumjs/mpt": "6.2.2",
     "@ethereumjs/tx": "5.4.0",
     "@ethereumjs/util": "9.1.0",
-    "@ethereumjs/verkle": "^0.1.0",
```
Comment (holgerd77) [inline packages/client/package.json:70]: Interesting! 🤯

File: packages/common/package.json [hunk 2]
```diff
@@ -57,8 +57,7 @@
     "tsc": "../../config/cli/ts-compile.sh"
   },
   "dependencies": {
-    "@ethereumjs/util": "^9.1.0",
-    "ethereum-cryptography": "^3.0.0"
```
Comment (holgerd77) [inline packages/common/package.json:61]: Another 🤯

File: packages/evm/package.json [hunk 3]
```diff
@@ -58,7 +58,6 @@
   "dependencies": {
     "@ethereumjs/common": "^4.4.0",
     "@ethereumjs/statemanager": "^2.4.0",
-    "@ethereumjs/tx": "^5.4.0",
```
Comment (holgerd77) [inline packages/evm/package.json:61]: And the third 🤯 😂

**Review by holgerd77 — APPROVED** [review]
Yeah, also: LGTM, super great clean-up, love this! ❤️ 😄

**Pull request merged**